package-lock.json

```diff
@@ -19,6 +19,7 @@
         "mockman-js": "^1.0.5",
         "react": "^17.0.2",
         "react-dom": "^17.0.2",
+        "react-router-dom": "^6.3.0",
```

Comment:
you have two lock files, yarn.lock and package.lock,

this is because, you've used yarn and npm to install packages, or it could be because when you installed the mockbee template the files were installed using yarn, and after that you might have used npm to install packages,

the solution would be to delete the node modules and both the .lock files and then run npm install or yarn install, to install it using one of the package managers.

Reply:
Thanks for pointing this out.